tests/forte/notebook_tutorial/handling_structued_data_test.py

```diff
@@ -0,0 +1,110 @@
+"""This module tests notebook handling_structued_data.ipynb ."""
+import os
+import unittest
+
+from forte.data.data_pack import DataPack
+from forte.pipeline import Pipeline
+from ft.onto.base_ontology import Token
+from forte.data.data_pack import DataPack
+from forte.pipeline import Pipeline
+from ft.onto.base_ontology import (
+    Token,
+    Sentence,
+    Document,
+)
+from forte.data.readers import OntonotesReader
+from forte.data.data_pack import DataPack
+from forte.pipeline import Pipeline
+
+
+class TestHandlingStructuedData(unittest.TestCase):
```

Comment:
Looks like we are rewriting the notebook code to do these tests. These are enough for our purposes (maybe). But there are a few issues with the approaches:
1. The work to get one notebook done is very long
2. If we change the notebook, we will need to change the dependent code again.